libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/bridge/SmartStoreReactBridge.java

```diff
@@ -285,21 +284,42 @@ public void upsertSoupEntries(ReadableMap args,
 	 * @return
 	 */
 	@ReactMethod
-	public void registerSoup(ReadableMap args,
-                             final Callback successCallback, final Callback errorCallback){
+	public void registerSoup(ReadableMap args, final Callback successCallback,
```

Comment:
Reworked this method to support the new arguments. Same as the ```Cordova``` plugin implementation.